src/SprykerEco/Zed/Payone/Business/Distributor/OrderPriceDistributorInterface.php

```diff
@@ -12,9 +12,9 @@
 interface OrderPriceDistributorInterface
 {
     /**
-     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param \Generated\Shared\Transfer\OrderTransfer|null $orderTransfer
      *
      * @return \Generated\Shared\Transfer\OrderTransfer
      */
-    public function distributeOrderPrice(OrderTransfer $orderTransfer): OrderTransfer;
+    public function distributeOrderPrice(?OrderTransfer $orderTransfer): OrderTransfer;
```

Comment:
if you are going to distribute order price that means **order is provided**
it cannot be nullable

Reply:
Fixed.